src/challenges.js

```diff
@@ -27,17 +44,17 @@ const cacheFunction = cb => {
 /* eslint-enable no-unused-vars */
 
 /* ======================== Recursion Practice ============================ */
-const reverseStr = str => {
+const reverseStr = (str) => {
   // reverse str takes in a string and returns that string in reversed order
   // The only difference between the way you've solved this before and now is that you need to do it recursivley!
 };
 
-const checkMatchingLeaves = obj => {
+const checkMatchingLeaves = (obj) => {
   // return true if every property on `obj` is the same
   // otherwise return false
 };
 
-const flatten = elements => {
+const flatten = (elements) => {
```

Comment:
both:
`-const flatten = elements => {`
and
`-const flatten = (elements) => {` 
should work with arrow functions when we only pass one argument to a function.